scripts/signing_secure_boot_prod.sh

```diff
@@ -0,0 +1 @@
+#In this script Vendor should code  the logic to build a secure boot image by using vendor flows
```

Comment:
this looks like placeholder, i do not know how to deal with it. why is this actually a placeholder.

Reply:
This script is a placeholder to give modularity to the vendor for the signing flow. 
The vendor can save in this script his own signing flow, signing in the same build server or external sign server, and can decide which sign tools to use.

Details explained in the HLD: 
https://github.com/sonic-net/SONiC/blob/6ec54974b9814b49e5acef1dc544b51d91198424/doc/secure_boot/hld_secure_boot.md#ArchitectureDesign